add new snipet vue

bobbyrahmanda13 opened the pull request:
Sorry, I really can't provide a good commit so please help

OkelleyDevelopment commented:
Hi @bobbyrahmanda13, some pointers to help get this PR into shape: 
- Before raising a PR please group the commits together with an interactive rebase (`git rebase -i <hash>`) as this makes it a little more clear on the changes. 
- The PR title should also match the changes (`add new snippet vue` doesn't necessarily imply the rename of the snippets that I also see).
- If you plan to have significant changes, the individual commit messages can be light on words so long as the description explains *why* it's needed -- since it's unreasonable to ask any sole person to know all languages, I do rely on the context from others to inform my reviews (plus the documentation that I reference).


If you'd like some help/clarifications, please ping me and I'll see how I can assist further :slightly_smiling_face:

bobbyrahmanda13 commented:
for naming commits or whatever honestly I'm a bad person at that, so I'm sorry for naming please help me

in this change I added snippets that will be used by Vue developers to speed up coding, currently Vue is developing very fast and there are even some new ones which I think if they use them it will help,

then for the javascript and typescript for the snippet I took it from the JS and ts snippets in this friendly snippet and I moved it to vue and some additions

I moved it because some people use this
```
broadnipLoader.lazy_load()
broadnip.filetype_extend("javascript", {"vue"})
broadnip.filetype_extend("typescript", {"javascript"})
broadnip.filetype_extend("vue", {"javascript","typescript"})
```
This makes the snippets in JavaScript and Typescript merge, and this makes them double
so I moved it to script.json so they don't use it anymore so it's just vue without the need to extend javascript and typescript,

because in javascript there is a console.log snippet with the prefix "cl" and in typescript the prefix is ​​"log" for console.log

Please correct me and ask for help if I'm wrong, please criticize so that Vue can be better
